bindings/python/pymongoarrow/types.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2021-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from datetime import datetime
+import enum
+
+from bson import Int64
+
+from pyarrow import timestamp, float64, int64, int32
+from pyarrow import DataType as _ArrowDataType
+import pyarrow.types as _atypes
+
+
+class _BsonArrowTypes(enum.Enum):
+    datetime = 1
+    double = 2
+    int32 = 3
+    int64 = 4
+
+
+_SUPPORTED_TYPE_IDENTIFIERS = (Int64, float, int, datetime)
```

Comment:
Is this needed? It seems like we can use _TYPE_NORMALIZER_FACTORY instead, eg:
```
typeid in _TYPE_NORMALIZER_FACTORY
```

Reply:
Good point. Done.